hpcgap/lib/oper.g

```diff
@@ -1276,6 +1276,106 @@ BIND_GLOBAL( "DeclareAttribute", function ( arg )
 end );
 
 
+##############################################################################
+##
+##  This code adds thread-local attributes, which are used in HPC-GAP.
+##  These are not complete replacements for normal attributes.
+##  In particular, they are not filters, so cannot be used in method selection.
```

Comment:
It strikes me that you say a lot about what they are not, but nothing at all about what they *are*. Of course one may assume that the reader will guess this from the name. The first sentence also seems to be written in reply to an false believe we presume the reader may harbor... 

How about this: "Thread local attributes are meant to work similar to usual attributes, but in a a thread-local fashion, i.e. for a single object <obj>, and a TLS attribute <attr>, each thread may store a different value for <attr> in <obj>. 
However, TLS attributes are not complete replacements for normal attributes. [...]"